src/ML/include/BipedalLocomotion/ML/velMANN.h

```diff
@@ -0,0 +1,169 @@
+/**
+ * @file velMANN.h
+ * @authors Paolo Maria Viceconte, Giulio Romualdi, Evelyn D'Elia
+ * @copyright 2023 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the BSD-3-Clause license.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_ML_VEL_MANN_H
+#define BIPEDAL_LOCOMOTION_ML_VEL_MANN_H
+
+#include <Eigen/Dense>
+#include <manif/SE2.h>
+
+#include <memory>
+
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+#include <BipedalLocomotion/System/Advanceable.h>
+#include <BipedalLocomotion/System/VariablesHandler.h>
+
+namespace BipedalLocomotion
+{
+namespace ML
+{
+
+/**
+ * MANNInput contains the input to the MANN network.
```

Comment:
This part of the documentation should be updated

Reply:
Updated in 75b78511.